src/geo_templates/default/main_page/front_page.tpl

```diff
@@ -205,18 +205,18 @@
 						</div>
 					</div>
 				<!-- END FEATURED CAROUSEL -->
-						
+
 				<!-- BANNER BEGIN -->
 				{* Example place for 300x100 image banners *}
-				<div class="content_box cntr banner">	         
-					<a href="http://www.kqzyfj.com/click-7066924-11191670" target="_blank"><img src="{external file='images/banners/1and1_300x100.jpg'}" alt="1and1 Hosting" title="1and1 Hosting" /></a>
-					<a href="http://www.hostmonster.com/track/geodesic/resources" target="_blank"><img src="{external file='images/banners/hostmonster_300x100.jpg'}" alt="HostMonster Hosting" title="HostMonster Hosting" /></a>
-					<a href="http://lunarpages.com/id/geodesic/goto/basic" target="_blank"><img src="{external file='images/banners/lunarpages_300x100.jpg'}" alt="Lunarpages.com Hosting" title="Lunarpages.com Hosting" /></a>
+                {* If you change the name of the sample banner below, be sure to update \admin\getting_started_checks\design_removeDefaultBannersFrontPage.php *}
+                {* NOTE: The above only pertains to Coders changing the software distro, not end users. *}

```

Comment:
These don't see changes very often, and when we do change things, it tends to be a huge re-design.

What do you think of leaving the comments out to reduce the extra info to someone trying to change the design?

Reply:
I put them for the reason that a previous coder changed the template but did not update getting started checker. So, the getting started checker for these items was malfunctioning with v18 and maybe long before.